all-in-one/jaeger-all-in-one-template.yml

```diff
@@ -15,7 +15,7 @@
 apiVersion: v1
 kind: List
 items:
-- apiVersion: extensions/v1beta1
+- apiVersion: apps/v1
```

Comment:
Would this cause failing in older k8s versions? Is `extensions/v1beta1` supported in k8s 1.16?